### PR TITLE
Add documentation for Onfido Connector

### PR DESCRIPTION
**Pull request by AfraHussaindeen**
## Purpose

- $Subject

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    secure = "false"
+    http_method = "POST"
+
+    [tenant_context]
+    enable_tenant_qualified_urls = "true"
+    enable_tenanted_sessions = "true"
+    rewrite.custom_webapps = ["/idv/onfido/"]
+    ```
+
+## Configuring Onfido Identity Verification Provider in WSO2 Identity Server Console
```
Comment (AfraHussaindeen) [inline docs/config.md:59]: The contents related to configuring onfido in the console needs to be changed according to the updated UI.

Comment (AfraHussaindeen) [reply docs/config.md:59]: Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/11/commits/e366865a038d05794bd3ffdaf1dbdce621a239c0

File: docs/config.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+1. Log in to the WSO2 Identity Server console using your admin credentials.
+2. In the WSO2 Identity Server Console, navigate to **Identity Verification Providers**.
+3. Click **+ New Identity Verification Provider** to create a new Identity Verification Provider (IDVP).
+4. You will be redirected to a page displaying a set of available IDVPs.
```
Comment (JayaShakthi97) [inline docs/config.md]:
```suggestion
4. You will be redirected to a page displaying a set of available IDVP templates.
```

Comment (AfraHussaindeen) [reply docs/config.md]: Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/11/commits/b4b34b108a78335be5a1a5cd4675cbeb6863e1cd

**Review by ashensw — APPROVED** [review]

**Pull request merged**